tests/output/test_io_backward_compat.py

```diff
@@ -273,6 +274,9 @@ def test_io_backward_compat_timestepping_save(version):
 @pytest.mark.parallel(nprocs=4)
 @pytest.mark.parametrize('version', ["2024_01_27"])
 def test_io_backward_compat_timestepping_load(version):
+    if DEFAULT_PARTITIONER != "chaco":
+        # This skip needs proper investigation
+        pytest.skip(reason="Backwards compatibility requires default partitioner to be chaco")
```

Comment:
There should be no distribution dependence in these tests. More directly, https://github.com/firedrakeproject/firedrake/pull/3483/files shows that these tests do not depend on chaco. It is likely that these tests failing is just a consequence of other functionalities used in these tests being broken.

Reply:
@ksagiyam can you confirm whether this now works without chaco?

Reply:
Merely removing chaco did not break tests (except `tests/multigrid/test_poisson_gmg.py::test_poisson_gmg_parallel_mg`) (please see the above link).

Could you just remove lines 277-279 and run CI, so that everyone can see the error (if any) in tomorrow's meeting?

Reply:
Thanks, I am testing again here after the rebase: https://github.com/firedrakeproject/firedrake/pull/3566. I wanted to double check what you had observed.

Reply:
It works, I will cherry-pick `482183b`